.bazelrc

```diff
@@ -6,6 +6,7 @@
 
 # Cache action outputs on disk so they persist across output_base and bazel shutdown (eg. changing branches)
 build --disk_cache=~/.cache/bazel-disk-cache
+build --workspace_status_command "/bin/bash bazel/workspace_status.sh"
```

Comment:
Please break this PR down further. Can you send a single PR adding the workspace status stuff? I'd like it in the `internal` repo as well. Can you take care of that as well?